index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <title>Form</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+
+</head>
+
+<body>
+  <div class="top">
+    <h1>freeCodeCamp Survey Form</h1>
+    <p><em>Thank you for taking the time to help us improve the platform</em></p>
+  </div>
+
+  <div class="main-form">
+    <form class="" action="/" method="post">
+      <div class="entry">
+        <label for="">Name</label><br>
+        <input class="name" type="text" name="name" placeholder="Enter Your Name">
+      </div>
+
+      <div class="entry">
+        <label for="">Email</label><br>
+        <input class="email" type="email" name="email" placeholder="Enter Your Email">
+      </div>
+
+      <div class="entry">
+        <label for="">Age(Optional)</label><br>
+        <input class="age" type="text" name="age" placeholder="Age">
+      </div>
+
+      <div class="entry">
+        <label for="">Which option best describes your current role?</label><br>
+        <select class="roles" name="roles" id="roles">
+          <option value="" disabled selected>Select current role</option>
+          <option >Student</option>
+          <option >Full time job</option>
+          <option >Full time learner</option>
+          <option >Prefer not to say</option>
+          <option >None</option>
+        </select>
+      </div>
+
+      <div class="entry">
+        <label for="">Would you recommend freeCodeCamp to a friend?</label> <br>
+        <input type="radio" checked>
+        <label for="def"> Definitely </label><br>
+        <input type="radio">
+        <label for="def"> Maybe </label><br>
+        <input type="radio">
+        <label for="def"> Not Sure </label><br>
+      </div>
```

Comment:
@AbbyNyakara! You're doing great with your review, keep it up. However, I got something to point out. 
- Your feedback is good but is lacking one key thing which is "The why".
When you propose a change, you should also tell why you are proposing the change.
Kindly try improving your feedback to have that ingredient, and you're good to go.

Reply:
@AbbyNyakara! You're doing great with your review, keep it up. However, I got something to point out. 
- Your feedback is good but is lacking one key thing which is "The why".
When you propose a change, you should also tell why you are proposing the change.
Kindly try improving your feedback to have that ingredient, and you're good to go.

Reply:
For this highlighted part, you can also employ the use of semantic tags because this better displays the relationship between different parts of your code. For this part for instance, you can use the section tag instead of a div. Please read more about semantic tags [here](https://github.com/microverseinc/curriculum-html-css/blob/main/html5.md).